tests/Fixer/Semicolon/MultilineWhitespaceBeforeSemicolonsFixerTest.php

```diff
@@ -424,6 +424,66 @@ public function provideSemicolonForChainedCallsFixCases()
                         ->method2();
                 ?>',
             ],
+            [
+                '<?php
+
+                    function foo($bar)
+                    {
+                        if ($bar === 1) {
+                            $baz
+                                ->bar()
+                            ;
+                        }
+
+                        return (new Foo($bar))
+                            ->baz()
+                        ;
+                    }
+;
+                ?>',
+                '<?php
+
+                    function foo($bar)
+                    {
+                        if ($bar === 1) {
+                            $baz
+                                ->bar();
+                        }
+
+                        return (new Foo($bar))
+                            ->baz();
+                    }
+;
+                ?>',
+            ],
+            [
+                '<?php
+
+                    $foo = (new Foo($bar))
+                        ->baz()
+                    ;
+
+                    function foo($bar)
+                    {
+                        $foo = (new Foo($bar))
+                            ->baz()
+                        ;
+                    }
+;
+                ?>',
+                '<?php
+
+                    $foo = (new Foo($bar))
+                        ->baz();
+
+                    function foo($bar)
+                    {
+                        $foo = (new Foo($bar))
+                            ->baz();
+                    }
+;
```

Comment:
Are those `;` at the end of new test cases intended?

Reply:
actually not! :) removed it :)